host_cleanup.sh

```diff
@@ -6,13 +6,13 @@ source lib/common.sh
 
 # Kill and remove the running ironic containers
 for name in ironic ironic-inspector dnsmasq httpd mariadb; do
-    sudo podman ps | grep -w "$name$" && sudo podman kill $name
-    sudo podman ps --all | grep -w "$name$" && sudo podman rm $name -f
+    sudo ${CONTAINER_RUNTIME} ps | grep -w "$name$" && sudo ${CONTAINER_RUNTIME} kill $name
+    sudo ${CONTAINER_RUNTIME} ps --all | grep -w "$name$" && sudo ${CONTAINER_RUNTIME} rm $name -f
 done
 
 # Remove existing pod
-if  sudo podman pod exists ironic-pod ; then
-    sudo podman pod rm ironic-pod -f
+if  sudo ${CONTAINER_RUNTIME} pod exists ironic-pod ; then
+    sudo ${CONTAINER_RUNTIME} pod rm ironic-pod -f
```

Comment:
Should this be run with podman only like pod creation?

Reply:
Agreed with @russellb .

Reply:
fixed, thank you